apiserver/gui.go

```diff
@@ -354,7 +364,7 @@ func (h *guiHandler) serveConfig(w http.ResponseWriter, req *http.Request) {
 	w.Header().Set("Content-Type", jsMimeType)
 	tmpl := filepath.Join(h.rootDir, "templates", "config.js.go")
 	if err := renderGUITemplate(w, tmpl, map[string]interface{}{
-		"base":             h.baseURLPath,
+		"base":             "/gui/",
```

Comment:
The base path used to contain the model uuid. The GUI appends the new model path to this and we end up with a URL where the last bit is ignored. By making the base path just be "/gui/", the GUI then appends the new model path bit and we end up with a properly formed URL that Juju recognises.